CMakeLists.txt

```diff
@@ -167,6 +167,7 @@ if( BUILD_WITH_TENSILE )
   set( Tensile_ARCHITECTURE "all" CACHE STRING "Tensile to use which architecture?")
   set( Tensile_CODE_OBJECT_VERSION "V2" CACHE STRING "Tensile code_object_version")
   set( Tensile_COMPILER "hcc" CACHE STRING "Tensile compiler")
+  set( Tensile_LIBRARY_FORMAT "yaml" CACHE STRING "Tensile library format")
 
```

Comment:
Thank you for this fix. The default for Tensile_LIBRARY_FORMAT is msgpack. An alternative is yaml, but the default is msgpack. Can you change this from yaml to msgpack.  Thank you.

Reply:
Thanks, it's fixed